am-lib/src/test/java/uk/gov/hmcts/reform/amlib/helpers/TestDataFactory.java

```diff
@@ -35,34 +36,42 @@ public static ExplicitAccessGrant createGrantForWholeDocument(String resourceId,
     public static ExplicitAccessGrant createGrantForWholeDocument(String resourceId,
                                                                   String accessorId,
                                                                   Set<Permission> permissions) {
-        return createGrant(resourceId, ImmutableSet.of(accessorId),
-            createPermissionsForWholeDocument(permissions));
+        return createGrant(resourceId, accessorId, createPermissionsForWholeDocument(permissions));
     }
 
     public static ExplicitAccessGrant createGrantForWholeDocument(String resourceId,
                                                                   Set<String> accessorId,
```

Comment:
Just noticed that it should be renamed to `accessorIds`

Reply:
👍